src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,222 @@
+""" Automation of proxy provider suite
+
+:requirement: IDM-SSSD-REQ : Proxy Provider
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import subprocess
+import ldap
+from sssd.testlib.common.utils import sssdTools, \
+    SSHClient, LdapOperations
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.usefixtures('setup_sssd_krb',
+                         'create_posix_usersgroups',
+                         'sssdproxyldap',
+                         'install_nslcd',
+                         'misc_test',
+                         'netgroups')
+@pytest.mark.tier1_3
+class TestProxyMisc(object):
+    """
+    This is test case class for proxy provider suite
+    """
+    def test_bz1036758(self, multihost, backupsssdconf):
+        """
+        :title: Allow for custom attributes in RDN bz1036758
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1036758
+        :id: 10eb49a4-b252-11ec-87ea-845cf3eff344
+        :steps:
+          1. Add a local user
+          2. sssd to fetch id information from local user
+          3. Auth should work using ldap provider
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+        """
+        client_e = multihost.client[0].ip
+        tools = sssdTools(multihost.client[0])
+        tools.clear_sssd_cache()
+        # Auth should work using ldap provider
+        ssh1 = SSHClient(client_e, username="foo1", password="Secret123")
+        ssh1.close()
```

Comment:
I'm not sure what the checks below should do, especially since no results are checked. The `chown` tests are in the original test as well but here as well I'm not sure why they are there.

The `su` test in the original test is different because it is used with the password, so I would assume the originaly su is called twice, first as root which always works if the user exists (this is what is done here) and then a second time as the user itself so that the password is requested. But since authentication is already tested with ssh it is not needed imo.

Reply:
changed